[2025-04-28T06:53:37.153+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Airflow_ETL_DAG.insert_viruses scheduled__2025-03-10T00:00:00+00:00 [queued]>
[2025-04-28T06:53:37.162+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Airflow_ETL_DAG.insert_viruses scheduled__2025-03-10T00:00:00+00:00 [queued]>
[2025-04-28T06:53:37.163+0000] {taskinstance.py:1361} INFO - Starting attempt 2 of 3
[2025-04-28T06:53:37.203+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): insert_viruses> on 2025-03-10 00:00:00+00:00
[2025-04-28T06:53:37.215+0000] {standard_task_runner.py:57} INFO - Started process 563 to run task
[2025-04-28T06:53:37.219+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'Airflow_ETL_DAG', 'insert_viruses', 'scheduled__2025-03-10T00:00:00+00:00', '--job-id', '13', '--raw', '--subdir', 'DAGS_FOLDER/airflow_etl.py', '--cfg-path', '/tmp/tmpvciddu12']
[2025-04-28T06:53:37.223+0000] {standard_task_runner.py:85} INFO - Job 13: Subtask insert_viruses
[2025-04-28T06:53:37.292+0000] {task_command.py:416} INFO - Running <TaskInstance: Airflow_ETL_DAG.insert_viruses scheduled__2025-03-10T00:00:00+00:00 [running]> on host 77b690a9fde8
[2025-04-28T06:53:37.426+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='Airflow_ETL_DAG' AIRFLOW_CTX_TASK_ID='insert_viruses' AIRFLOW_CTX_EXECUTION_DATE='2025-03-10T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-10T00:00:00+00:00'
[2025-04-28T06:53:37.469+0000] {logging_mixin.py:151} INFO - Insertion des virus termin√©e.
[2025-04-28T06:53:37.470+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-04-28T06:53:37.484+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=Airflow_ETL_DAG, task_id=insert_viruses, execution_date=20250310T000000, start_date=20250428T065337, end_date=20250428T065337
[2025-04-28T06:53:37.594+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-04-28T06:53:37.621+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
